mxcubeweb/core/components/lims.py

```diff
@@ -323,3 +323,30 @@ def synch_with_lims(self, lims_name):
                 self.sample_list_sync_sample(sample_info)
 
         return self.sample_list_get()
+
+    def get_lims_samples(self, lims_name: str) -> dict:
```

Comment:
I think we can make the comment or doc string clearer but mentioning that this only returns LIMS samples and NOT any other samples.